GitUI/CommandsDialogs/SettingsDialog/Pages/DiffViewerSettingsPage.Designer.cs

```diff
@@ -51,6 +52,7 @@ private void InitializeComponent()
             this.tableLayoutPanelForDiffViewer.ColumnStyles.Add(new System.Windows.Forms.ColumnStyle());
             this.tableLayoutPanelForDiffViewer.ColumnStyles.Add(new System.Windows.Forms.ColumnStyle());
             this.tableLayoutPanelForDiffViewer.Controls.Add(this.chkShowDiffForAllParents, 0, 6);
+            this.tableLayoutPanelForDiffViewer.Controls.Add(this.useDifftoolDirDiff, 0, 8);
```

Comment:
This is not really related to the diff viewer, but there are not really better place for it. Advanced/Detailed need to be rearranged...

Reply:
Agreed, feels weird here, but didn't change.